system/hardware/power_monitoring.py

```diff
@@ -111,12 +111,18 @@ def should_shutdown(self, ignition: bool, in_car: bool, offroad_timestamp: float
     if offroad_timestamp is None:
       return False
 
+    # Max Time Offroad
+    try:
+      MAX_TIME_OFFROAD_SP = int(self.params.get("MaxTimeOffroad", encoding="utf8")) * 60
+    except Exception:
+      MAX_TIME_OFFROAD_SP = MAX_TIME_OFFROAD_S
```

Comment:
- Avoid using capitalized variable names like `MAX_TIME_OFFROAD_SP` for non-constants. Consider making it a class attribute instead.
- This logic block would be better encapsulated in a helper method.